### ISSUE TITLE
Scroll při tažení strany myší

### ISSUE THREAD

**Issue by luckajirku** [proarc/proarc-client#190]
Ve stávajícím rozhraní při popisu/editaci stran, když mám označenou stranu a táhnu ji někam, lze zároveň scrollovat. U nového to nejde, tam scroll přestává fungovat ve chvíli, kdy nějakou stranu uchopím. Bylo by fajn to vrátit, u větších dávek je to docela praktická věc.

**Comment by albertoh** [comment]
Pri posouvani oznacene strany scroll se provadi automaticky, jakmile se posunete na vrchni/dolni kraj.
Aktivovat scroll koleckem mysi nejde

**Comment by albertoh** [comment]
Problem je globalne v javascriptu. Takzvane drag and drop.
V novem klientovi pri posouvani je videt ten objekt (obrazek, popis) jak vidime nez zacneme ho posouvat. To je primo podporavano v javascriptu.
Ve stavajcim klientovi misto toho se vytvari novy element (cervanou lamanou carku).
Opravit chovani by bylo casove narocne.

**Comment by ZdenkaSera** [comment]
@luckajirku Prosím, je to tohle kritické nebo to můžeme a) zavřít, b) odložit na případné podzimní vylepšení? Děkuji.

**Comment by luckajirku** [comment]
Kritické to není, můžeme to odložit (ale úplně bych to teď nezavírala).

**Comment by ZdenkaSera** [comment]
Protože se neobjevil žádný komentář v souvislosti s přiřazením štítku "k realizaci", odebírám štítek "návrh na rozšíření".

**Comment by albertoh** [comment]
Z dokumentace javascriptu:

"Unfortunately, if you're using the [Drag and Drop API](https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API) with the draggable='true' attribute, wheel and scroll events are not fired during a drag, so you will be unable to fire and handle a scroll event."

Vsude pouzivame HTML_Drag_and_Drop_API

**Comment by ZdenkaSera** [comment]
@albertoh Alberto, prosím, co z toho plyne - že je tedy tento požadavek nerealizovatelný? Děkuji.

**Comment by albertoh** [comment]
Je realizovatelne, ale casove narocne

**Comment by luckajirku** [comment]
Nerozumím tomu, proč to vypadlo z realizace. V tabulce návrhů 2022 v hlasování mělo tohle issue nejvíc bodů (27), tak proč o tom znovu hlasujeme pro další rok?

**Comment by ZdenkaSera** [comment]
Zařadila jsem to znovu vzhledem k poslednímu Albertovu vyjádření, že je to časově náročné a vzhledem k tomu, jak dlouho nám to tu leží. Mám to ale označené "schůzka" abych se na to nezapomněla @albertoh  zeptat.

**Comment by luckajirku** [comment]
@albertoh je tohle pořád někde k testu? pokud to má být stále na masteru, tak tam to tak, jak jsme chtěli, nefunguje.

**Comment by albertoh** [comment]
Jo. Je to k testovani. 
Co nefunguje? Ta oblast, kde zacina scrollovani by mela byt vetsi,. Muzu ji delat jeste vetsi, kdyz tak

Jak jsem psal vise, nejde pouzivat scroll mysi. Minimalne nejde bez uplneho prepsani zpusobu posouvani.

**Comment by luckajirku** [comment]
No, co jsi psal výše*, dávalo trochu více naděje (viz níže - jen pro pořádek:-)), ale OK, jestli je to tak, tak asi nemá smysl, aby to   bylo jako k testování, když to, kvůli čemu to bylo založené, k testování není. Pokud se nedohodneme, že se to má udělat, tak se to issue může dát jako odložené, nebo i úplně zavřít s tím, že se realizovat nebude. Ať to tady zbytečně nevisí.  
*_14. 6. 2022: "Opravit chovani by bylo casove narocne."
14. 8. 2023: "Je realizovatelne, ale casove narocne"_

**Comment by albertoh** [comment]
A nestaci zvetsit oblasti automatickeho scrollovani?

**Comment by luckajirku** [comment]
Nestačí. Ale jak říkám, můžeme to odložit/zrušit, ať je čas na ty další věci, které čekají.
